specification/resources/resource-manager/Microsoft.Resources/preview/2019-10-01-preview/deploymentScripts.json

```diff
@@ -670,6 +675,32 @@
           "description": "Container group name, if not specified then the name will get auto-generated. Not specifying a 'containerGroupName' indicates the system to generate a unique name which might end up flagging an Azure Policy as non-compliant. Use 'containerGroupName' when you have an Azure Policy that expects a specific naming convention or when you want to fully control the name. 'containerGroupName' property must be between 1 and 63 characters long, must contain only lowercase letters, numbers, and dashes and it cannot start or end with a dash and consecutive dashes are not allowed. To specify a 'containerGroupName', add the following object to properties: { \"containerSettings\": { \"containerGroupName\": \"contoso-container\" } }. If you do not want to specify a 'containerGroupName' then do not add 'containerSettings' property.",
           "minLength": 1,
           "maxLength": 63
+        },
+        "restartPolicy": {
```

Comment:
It is the same setting actually. In a recent PR update this property got removed (the RP got updated to not return this value, since it is read-only and sets only one value)